Added in static files for HeronUI so that Offline Heron Deployment works

mjschmidt opened the pull request.

cckellogg commented:
Can we add versions to the  js file names? 

Also, there a few more dependencies:
//cdnjs.cloudflare.com/ajax/libs/d3/3.4.11/d3.min.js
//cdnjs.cloudflare.com/ajax/libs/d3-tip/0.6.3/d3-tip.min.js

See files:
config.html
exception.html
topologies.html
topology.html

mjschmidt commented:
I added the js files from the links they were being called from, added the files names and replaced the cloudera links in:
application.html
config.html
exception.html
topologies.html
topology.html

mjschmidt commented:
YAY it worked (:

kramasamy commented:
Awesome!

Sent from my iPhone

> On Nov 2, 2017, at 8:29 AM, Michael Schmidt <notifications@github.com> wrote:
> 
> YAY it worked (:
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>

cckellogg commented on heron/tools/ui/resources/templates/application.html:
```diff
@@ -15,8 +15,8 @@
-    <script src="//cdnjs.cloudflare.com/ajax/libs/react/0.10.0/react.js"></script>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/react/0.10.0/JSXTransformer.js"></script>
+    <script src="{{ static_url('js/react.0.10.0.js') }}"></script>
+    <script src="{{ static_url('js/JSXTransformer-0.10.0.jss') }}"></script>
```
spelling mistake? should just be .js right?
